File: scratch/fetch_config.js
require('dotenv').config();
const { createClient } = require('@supabase/supabase-js');
const supabase = createClient(process.env.SUPABASE_URL, process.env.SUPABASE_ANON_KEY);

async function check() {
  const { data, error } = await supabase.from('platform_config').select('*').eq('key', 'platform_fee');
  console.log('PLATFORM FEE DB VALUE:', data);
}
check();

File: scratch/fetch_config.dart
import 'package:supabase/supabase.dart';
import 'package:flutter_dotenv/flutter_dotenv.dart';
import 'dart:io';

void main() async {
  dotenv.testLoad(fileInput: File('.env').readAsStringSync());
  final client = SupabaseClient(
    dotenv.env['SUPABASE_URL']!,
    dotenv.env['SUPABASE_ANON_KEY']!,
  );
  
  final res = await client.from('platform_config').select('*').eq('key', 'platform_fee');
  print('PLATFORM_FEE: $res');
  exit(0);
}
